VaticanBAV/et/BAVet196.xml

```diff
@@ -0,0 +1,308 @@
+                                </item>
+                              </list>
+                            </collation>
+                            <condition key="good">The lower margins show signs of being exposed to moisture.</condition>
+                          </supportDesc>
+
+                          <layoutDesc>
+                            <layout writtenLines="15" />
+                          </layoutDesc>
+                        </objectDesc>
+
+                        <decoDesc>
+                          <decoNote xml:id="d1" type="frame">
+                            <locus target="#3r"/>
+                            <desc>A decorative border with black and red lines.</desc>
+                          </decoNote>
+                        </decoDesc>
+
+                        <additions>
+                          <list>
+
+                            <item xml:id="a1">
+                              <locus target="#1v" />
+                              <desc type="Supplication">
+                                <title>ʾAsmāt Prayer</title>
+                              </desc>
+                              <q xml:lang="gez">
+                                <seg part="I">ትዮር፡ አላትዮር፡ ክትዮር፡ ፍክትዮር፡ በኃይለ፡ ዝንቱ፡ አስማቲከ፡ አንህዮ፡ ወአውርዶ፡ ለዝንቱ፡ በኖ፡ ዘሀሎ፡ ውስተ፡ ከርሱ፡ ለገብርከ፡ እገሌ፡</seg>
+                              </q>
+                            </item>
+
+                            <item xml:id="e1">
+                              <locus from="1r" to="2v" />
+                              <desc>Some scribbles in the margins in pen and pencil.</desc>
+                            </item>
+
+                          </list>
+                        </additions>
+
+                      </physDesc>
+
+                    </msPart>
+
+                    <msPart xml:id="p2">
+                      <msIdentifier><idno/></msIdentifier>
+
+                      <msContents>
+                        <summary/>
+
+                        <msItem xml:id="p2_i1">
+                          <locus from="55r" to="77v" />
+                          <title type="complete" ref="LIT2866RepCh147" />
+                          <textLang mainLang="gez" />
+                        </msItem>
+
+                        <msItem xml:id="p2_i2">
+                          <locus from="78r" to="80r" />
+                          <title>ʾAsmāt Prayer</title>
+                          <textLang mainLang="gez" />
+                          <incipit xml:lang="gez"><gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran" /> እስር፡ ጕርዔሆሙ፡ ለፀርየ፡ <gap reason="ellipsis" extent="unknown" resp="PRS4805Grebaut PRS9530Tisseran" /> በስመ፡ አብ፡ በል፡ ጸለየ፡ ምስለ፡ ዳዊት፡ ሳዶር፡ አላዶር፡ ዳናት፡ አዴራ፡ ሮዳስ፡</incipit>
+                          <note>Written in pencil.</note>
+                        </msItem>
+
+                        <msItem xml:id="p2_i3">
+                          <locus target="#80v" />
+                          <title/>
```

Comment:
I take that no more information is given in the catalogue? Maybe you could open a text identification issue.

Reply:
Ahh, sorry, I meant to back to it. Grébaut notes "A historical note, it seems, regarding the time of some calamity"